tracks/ruby/exercises/acronym/mentoring.md

```diff
@@ -11,20 +11,15 @@ end
 ```
 with variations for the regex (see below).
 
-
 ## Common suggestions
 * Most first submissions split the phrase into words with `String#split`: 
   ```ruby
   phrase.split(/[ -]/)
   ```
-  , followed by operations to find the first letters.
-  Variation: first `gsub` or `tr` the non-space delimiters, then `split`
+  followed by operations to find the first letters.
+  Variation: first `gsub` or `tr` the non-space delimiters, then `split`.
   
-In all these cases: point to String#scan.   
-* The first use of `scan` mostly is a scan for full words. That's a perfect moment to show the power of `scan`: 
-when it returns the first letters only, it will save a few operations.  
-* As this exercise is not about Regular Expressions, it's recommended to 
-give the regex away for free, especially if they already found `scan(/\w+/)`
+In all these cases: it's recommended to suggest the `String#scan` method and, because the exercise is not about Regular Expressions, give them also the RegEx (`/\b[a-zA-Z]/`). (At this point in the track, it's more about experiencing the strengths of Ruby's built-in methods then making them find them. There's ample opportunity for research later on in the track.)   
```

Comment:
Think there's a small typo here.

```suggestion
In all these cases: it's recommended to suggest the `String#scan` method and, because the exercise is not about Regular Expressions, give them also the RegEx (`/\b[a-zA-Z]/`). (At this point in the track, it's more about experiencing the strengths of Ruby's built-in methods than making them find them. There's ample opportunity for research later on in the track.)   
```